elements/nuxeo-workflow-graph/nuxeo-workflow-graph.html

```diff
@@ -264,7 +268,7 @@
       },
 
       _fetchGraph: function() {
-        if (this.workflowId && this.workflowId.length > 0) {
+        if (this.taskId && this.workflowId && this.workflowId.length > 0) {
```

Comment:
Same suggestion as in https://github.com/nuxeo/nuxeo-web-ui/pull/411#discussion_r241802990
```suggestion
        if (this.taskId && this.workflowId) {
```